gremlin-go/README.md

```diff
@@ -65,91 +58,222 @@ import (
 
 func main() {
 	// Creating the connection to the server.
-	driverRemoteConnection := gremlingo.NewDriverRemoteConnection("localhost", 8182)
+	driverRemoteConnection, err := gremlingo.NewDriverRemoteConnection("localhost", 8182)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
 	// Cleanup
-	defer driverRemoteConnection.Close()
-	// Submit a traversal (string format in milestone 1).
+	defer func(driverRemoteConnection *gremlingo.DriverRemoteConnection) {
+		err := driverRemoteConnection.Close()
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+	}(driverRemoteConnection)
+	
+	// Submit a simple string traversal
 	resultSet, err := driverRemoteConnection.Submit("1 + 1")
-	// Check for traversal execution errors, print them if there are any and exit.
 	if err != nil {
 		fmt.Println(err)
 		return
 	}
+	
 	// Grab the first result from all the results in the ResultSet.
 	result := resultSet.All()[0]
+	
 	// Print the first result.
-	fmt.Println(result.AsString())
+	fmt.Println(result.GetString())
 }
 ```
 
 Note: The exact import name as well as the module prefix for `NewDriverRemoteConnection` may change in the future.
 
+You may need to run `go mod tidy` to import the remaining dependencies of the `gremlin-go` driver, after which you should see an updated `go.mod` file:
```

Comment:
I think it would be better to list this part as a step after having imported the driver in the script, rather than worded in a way that implies its a failsafe.

Reply:
Ok